### PR TITLE
Integrate CN-DPM into Sequoia research tree

### PR DESCRIPTION
**Pull request by ryanlindeborg**

### CODE REVIEW

File: setup.py
Comment (lebrice) [inline setup.py:47]: One thing: I'm not entirely sure about the best way to allow people to install this cn_dpm "plugin":
- Having it as a submodule in Sequoia, and install it optionally like this with extras_require (which I'm not 100% sure would work)
- Having it as an external package, with its own setup.py (as is currently the case I think) and specify a "Method" entry_point, so that Sequoia can load it as a plugin, rather than it being included within Sequoia

In any case, you can leave this as-is, and we'll fix it later.

**Review by lebrice — APPROVED** [review]
Looks good here, I'll check out the CN-DPM repo next.
We can chat a bit about how best to make the method installable (either through Sequoia or though a standalone `sequoia.methods.cn_dpm` package) during our next meeting.

Thanks again!

**Comment by ryanlindeborg** [comment]
Hey @lebrice , thanks for the review. Once I merged master back into my branch, it automatically expired the old PR approval. Could you re-approve? I'll merge it after

**Review by lebrice — APPROVED** [review]

**Pull request merged**
